sparsity/sparse_frame.py

```diff
@@ -591,10 +591,13 @@ def drop_duplicate_idx(self, **kwargs):
     def __getitem__(self, item):
         if not isinstance(item, (tuple, list)):
             item = [item]
-        if item is not None and len(item) > 0:
+        if len(item) > 0:
```

Comment:
@kayibal Previously you took care of the situation when `None` is passed, now you don't. I'm not sure if it's necessary, just wanted to drop a hint.

Reply:
It should raise an error

Reply:
Now it's great.